src/rest_model.py

```diff
@@ -20,65 +20,92 @@
     'closed_at': fields.DateTime(dt_format='iso8601'),
     'creator_name': fields.String,
     'creator_url': fields.Url(description='Creator Url'),
+    'probable_cve': fields.Boolean()
 })
 
 
+class UnsanitizeStringFormat(fields.Raw):
+    """Abstract the input string formating."""
+
+    __schema_type__ = 'array'
+
+    def format(self, value):
+        """Unsanitizies the input string."""
+        return unsanitize(value)
+
+
 class ISO8601Format(fields.Raw):
     """Abstracts iso8601 string formatting."""
 
+    __schema_type__ = 'string'
+    __schema_example__ = '2020-02-23 19:02:15+00:00'
+
     def format(self, value):
         """Convert given epoch to iso8601 string."""
-        return to_date_str(value)
+        return None if value is None else to_date_str(value)
```

Comment:
`return to_date_str(value) if value else None` is more clear.

Reply:
done